lib/Timber.php

```diff
@@ -640,7 +659,6 @@ public static function fetch( $filenames, $data = array(), $expires = false, $ca
 	public static function render( $filenames, $data = array(), $expires = false, $cache_mode = Loader::CACHE_USE_DEFAULT ) {
 		$output = self::fetch($filenames, $data, $expires, $cache_mode);
```

Comment:
I think we just need to replace this with `self::compile` and that failing test will resolve